src/utils/makeSuspenseful.tsx

```diff
@@ -14,21 +13,18 @@ const suspenseSpinnerStyle = {
   height: '100vh',
 }
 
-export function makeSuspenseful<T extends FC<any>>(Component: T, spinnerStyle: BoxProps = {}) {
-  return (props: ComponentProps<T>) => {
-    const boxSpinnerStyle = useMemo(() => ({ ...suspenseSpinnerStyle, ...spinnerStyle }), [])
-
-    const suspenseSpinner = useMemo(
-      () => (
-        <Box sx={boxSpinnerStyle}>
-          <CircularProgress />
-        </Box>
-      ),
-      [boxSpinnerStyle],
```

Comment:
`spinnerStyle` was unused, removed.
Since that now has no component-level dependencies:

- removed `boxSpinnerStyle` in profit of `suspenseSpinnerStyle` only
- moved `suspenseFallback` to component vs. memoized element in HOC scope so we can export it for reuse